cmd/cli/commands/worker.go

```diff
@@ -244,3 +250,102 @@ var workerDebugStateCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var workerLogs = &cobra.Command{
+	Use:   "logs",
+	Short: "Subscribe for worker's logs",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := metadata.NewOutgoingContext(context.Background(), metadata.MD{
+			util.WorkerAddressHeader: []string{workerAddr},
+		})
+
+		cc, err := newClientConn(ctx)
+		if err != nil {
+			return err
+		}
+
+		steam, err := sonm.NewInspectClient(cc).WatchLogs(ctx, &sonm.InspectWatchLogsRequest{})
+		if err != nil {
+			return err
+		}
+
+		for {
+			message, err := steam.Recv()
+			if err != nil {
+				if err == io.EOF {
+					return nil
+				}
+
+				return err
+			}
+
+			cmd.Printf("%s", message.GetMessage())
+		}
+	},
+}
+
+var workerAddCapabilityCmd = &cobra.Command{
+	Use:   "add-capability ETH_ADDR CAPABILITY TTL",
```

Comment:
In other commands, we're describing required params as `<param>`.
I'd suggest allowing to set TTL as `time.Duration` which I think more user-friendly. BTW, can we use text IDs for the Capability instead of numbers?

Reply:
raw int produces nanosecods. Are you sure this is what we want?